src/store/redusers/get_data_reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { SET_DATA } from '../constants';
+
```

Comment:
please check the naming of the file and path.
Did you mean src/store/reducers/get-data-reducer ?

I also have some comments on the naming but I will share them a bit later when you will introduce few more actions and/or reducers.

Reply:
+